tests/zfs-tests/tests/functional/channel_program/synctask_core/tst.list_clones.ksh

```diff
@@ -27,8 +27,7 @@ log_assert "Listing zfs clones should work correctly."
 
 function cleanup
 {
-	datasetexists $TESTPOOL/$TESTFS@$TESTSNAP && \
-	    log_must zfs destroy -R $TESTPOOL/$TESTFS@$TESTSNAP
+	datasetexists $TESTPOOL/$TESTFS@$TESTSNAP "-R"
```

Comment:
If "-R" is passed to `datasetexists`, it should fail with invalid option 'R'
Perhaps you intended `destroy_dataset` instead?

Reply:
Indeed I did, thanks!